src/xref2/ref_tools.ml

```diff
@@ -139,14 +139,25 @@ let find find sg name =
   | None -> Error (`Find_by_name (`Any, name))
 
 let module_lookup_to_signature_lookup env (ref, cp, m) =
-  Tools.signature_of_module env m
+  let rec handle_expansion : Tools.expansion -> _ = function
```

Comment:
But we can't because we don't have an id. This means we can't resolve paths like this:

```ocaml
module X : functor ( Y : sig type t end ) -> sig
        module Z = Y
end

(** {!X.Z.t} *)
```

Reply:
This didn't work before either though, so let's leave this as is and we've got https://github.com/ocaml/odoc/issues/882 to remind us.